ui/Comments/BackButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { TouchableOpacity } from "react-native";
+import { ListItem } from "react-native-elements";
+
+type Props = {
+  onBack: () => void;
+};
+
+const BackButton = (props: Props) => {
+  const onBackHandler = () => {
+    props.onBack();
+  };
+
+  return (
+    <TouchableOpacity onPress={onBackHandler} activeOpacity={0.5}>
```

Comment:
Directly pass `props.onBack` into `TouchableOpacity` - no need to create a separate `onBackHandler` function.

Reply:
I thought it would increase readability and hence created separate functions for many one liner functions also. Should I remove them all or is this specific?

Reply:
A separate function increases readability if it represents a separate action. In this case, both `onBackHandler` and `props.onBack` mean the same thing - a function to be called for going back. Then a separate function is not necessary.